src/Patchwork/PHP/Shim/Mbstring.php

```diff
@@ -72,6 +72,14 @@ public static function mb_convert_encoding($s, $to_encoding, $from_encoding = IN
     {
         INF === $from_encoding and $from_encoding = self::$internal_encoding;
 
+        if (is_array($from_encoding)) {
+            $from_encoding = implode(',', $from_encoding);
```

Comment:
Instead of imploding, I'd rather explode when a coma is found, and pass the from enc to mb_detect_encoding if and only if it's an array